src/praisonai/praisonai/integration/bridges/kanban_bridge.py

```
@@ -0,0 +1,113 @@
+        
+        # Register startup handler if the backend supports it
+        if hasattr(backends, 'register_startup_handler'):
+            backends.register_startup_handler(_startup_handler)
+            log.debug("kanban dispatcher startup handler registered")
+        else:
+            # Fallback: try immediate start if event loop is running
+            try:
+                import asyncio
+                loop = asyncio.get_running_loop()
+                if loop and not is_dispatcher_running():
+                    asyncio.create_task(start_kanban_dispatcher())
```

Ruff (0.15.14)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)